Crash on tapping any episode

# Checklist
<!-- Place an x in the boxes to tick them: [x] -->

- [x] I have used the search function to see if someone else has already submitted the same bug report.
- [x] I will describe the problem with as much detail as possible.
- [x] If the bug only to occurs with a certain podcast, I will include the URL of that podcast.

# System info
<!-- The following information is very important to fill out because some bugs may only occur on certain devices or versions of Android. -->

**App version**: 1.8.1 (Google Play)
<!-- The latest version may be different depending on your device. You can find the version in AntennaPod's settings. -->

**Android version**: 5.1.1 (stock ROM)

**Device model**:
Model: PE-TL10
Device: hwPE
Product: PE-TL10

# Bug description

**Steps to reproduce**:
1. Open a subscribed podcast
2. Tap any episode

**Expected behaviour**:
I guess it shouldn't crash. I've just installed the app and never seen other behavior.

**Current behaviour**:
The app crashes

**First occurred**: (e.g. about x days/weeks ago)
Today (16.08.2020). But as I said I have no previous experience with the app.

**Environment**: 
Nothing unusual. Most of settings are default.

**Stacktrace/Logcat**:
<!-- If you are experiencing a crash, including the stacktrace will likely get it fixed sooner. AntennaPod has an `export logs` feature for this. -->
```
android.view.InflateException: Binary XML file line #166: Error inflating class android.webkit.WebView
	at android.view.LayoutInflater.createView(LayoutInflater.java:652)
	at com.android.internal.policy.impl.PhoneLayoutInflater.onCreateView(PhoneLayoutInflater.java:55)
	at com.android.internal.policy.impl.HwPhoneLayoutInflater.onCreateView(HwPhoneLayoutInflater.java:75)
	at android.view.LayoutInflater.onCreateView(LayoutInflater.java:701)
	at android.view.LayoutInflater.createViewFromTag(LayoutInflater.java:760)
	at android.view.LayoutInflater.rInflate(LayoutInflater.java:825)
	at android.view.LayoutInflater.inflate(LayoutInflater.java:523)
	at android.view.LayoutInflater.inflate(LayoutInflater.java:425)
	at de.danoeh.antennapod.fragment.ItemFragment.onCreateView(SourceFile:134)
	at androidx.fragment.app.Fragment.performCreateView(SourceFile:2600)
	at androidx.fragment.app.FragmentManagerImpl.moveToState(SourceFile:881)
	at androidx.fragment.app.FragmentManagerImpl.moveFragmentToExpectedState(SourceFile:1238)
	at androidx.fragment.app.BackStackRecord.executeOps(SourceFile:434)
	at androidx.fragment.app.FragmentManagerImpl.executeOps(SourceFile:2079)
	at androidx.fragment.app.FragmentManagerImpl.executeOpsTogether(SourceFile:1869)
	at androidx.fragment.app.FragmentManagerImpl.removeRedundantOperationsAndExecute(SourceFile:1824)
	at androidx.fragment.app.FragmentManagerImpl.execSingleAction(SourceFile:1696)
	at androidx.fragment.app.BackStackRecord.commitNowAllowingStateLoss(SourceFile:299)
	at androidx.fragment.app.FragmentStatePagerAdapter.finishUpdate(SourceFile:259)
	at androidx.viewpager.widget.ViewPager.populate(SourceFile:1244)
	at androidx.viewpager.widget.ViewPager.populate(SourceFile:1092)
	at androidx.viewpager.widget.ViewPager.onMeasure(SourceFile:1622)
	at android.view.View.measure(View.java:17609)
	at android.view.ViewGroup.measureChildWithMargins(ViewGroup.java:5547)
	at android.widget.FrameLayout.onMeasure(FrameLayout.java:436)
	at android.view.View.measure(View.java:17609)
	at android.widget.RelativeLayout.measureChildHorizontal(RelativeLayout.java:727)
	at android.widget.RelativeLayout.onMeasure(RelativeLayout.java:463)
	at android.view.View.measure(View.java:17609)
	at androidx.drawerlayout.widget.DrawerLayout.onMeasure(SourceFile:1119)
	at android.view.View.measure(View.java:17609)
	at android.view.ViewGroup.measureChildWithMargins(ViewGroup.java:5547)
	at android.widget.FrameLayout.onMeasure(FrameLayout.java:436)
	at androidx.appcompat.widget.ContentFrameLayout.onMeasure(SourceFile:143)
	at android.view.View.measure(View.java:17609)
	at android.view.ViewGroup.measureChildWithMargins(ViewGroup.java:5547)
	at android.widget.FrameLayout.onMeasure(FrameLayout.java:436)
	at android.view.View.measure(View.java:17609)
	at android.view.ViewGroup.measureChildWithMargins(ViewGroup.java:5547)
	at android.widget.FrameLayout.onMeasure(FrameLayout.java:436)
	at android.view.View.measure(View.java:17609)
	at android.view.ViewGroup.measureChildWithMargins(ViewGroup.java:5547)
	at android.widget.LinearLayout.measureChildBeforeLayout(LinearLayout.java:1467)
	at android.widget.LinearLayout.measureVertical(LinearLayout.java:747)
	at android.widget.LinearLayout.onMeasure(LinearLayout.java:638)
	at android.view.View.measure(View.java:17609)
	at android.view.ViewGroup.measureChildWithMargins(ViewGroup.java:5547)
	at android.widget.FrameLayout.onMeasure(FrameLayout.java:436)
	at com.android.internal.policy.impl.PhoneWindow$DecorView.onMeasure(PhoneWindow.java:2639)
	at android.view.View.measure(View.java:17609)
	at android.view.ViewRootImpl.performMeasure(ViewRootImpl.java:2082)
	at android.view.ViewRootImpl.measureHierarchy(ViewRootImpl.java:1217)
	at android.view.ViewRootImpl.performTraversals(ViewRootImpl.java:1423)
	at android.view.ViewRootImpl.doTraversal(ViewRootImpl.java:1105)
	at android.view.ViewRootImpl$TraversalRunnable.run(ViewRootImpl.java:6186)
	at android.view.Choreographer$CallbackRecord.run(Choreographer.java:793)
	at android.view.Choreographer.doCallbacks(Choreographer.java:606)
	at android.view.Choreographer.doFrame(Choreographer.java:575)
	at android.view.Choreographer$FrameDisplayEventReceiver.run(Choreographer.java:779)
	at android.os.Handler.handleCallback(Handler.java:739)
	at android.os.Handler.dispatchMessage(Handler.java:95)
	at android.os.Looper.loop(Looper.java:135)
	at android.app.ActivityThread.main(ActivityThread.java:5595)
	at java.lang.reflect.Method.invoke(Native Method)
	at java.lang.reflect.Method.invoke(Method.java:372)
	at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:960)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:755)
	Caused by: java.lang.reflect.InvocationTargetException
	at java.lang.reflect.Constructor.newInstance(Native Method)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:288)
	at android.view.LayoutInflater.createView(LayoutInflater.java:626)
	... 66 more
	Caused by: android.content.res.Resources$NotFoundException: String resource ID #0x3040005
	at android.content.res.HwResources.getText(HwResources.java:1314)
	at android.content.res.Resources.getString(Resources.java:399)
	at com.android.org.chromium.content.browser.ContentViewCore.setContainerView(ContentViewCore.java:733)
	at com.android.org.chromium.content.browser.ContentViewCore.initialize(ContentViewCore.java:657)
	at com.android.org.chromium.android_webview.AwContents.createAndInitializeContentViewCore(AwContents.java:649)
	at com.android.org.chromium.android_webview.AwContents.setNewAwContents(AwContents.java:798)
	at com.android.org.chromium.android_webview.AwContents.<init>(AwContents.java:634)
	at com.android.org.chromium.android_webview.AwContents.<init>(AwContents.java:571)
	at com.android.webview.chromium.WebViewChromium.initForReal(WebViewChromium.java:313)
	at com.android.webview.chromium.WebViewChromium.access$100(WebViewChromium.java:98)
	at com.android.webview.chromium.WebViewChromium$1.run(WebViewChromium.java:265)
	at com.android.webview.chromium.WebViewChromium$WebViewChromiumRunQueue.drainQueue(WebViewChromium.java:125)
	at com.android.webview.chromium.WebViewChromium$WebViewChromiumRunQueue$1.run(WebViewChromium.java:112)
	at com.android.org.chromium.base.ThreadUtils.runOnUiThread(ThreadUtils.java:144)
	at com.android.webview.chromium.WebViewChromium$WebViewChromiumRunQueue.addTask(WebViewChromium.java:109)
	at com.android.webview.chromium.WebViewChromium.init(WebViewChromium.java:262)
	at android.webkit.WebView.<init>(WebView.java:554)
	at android.webkit.WebView.<init>(WebView.java:489)
	at android.webkit.WebView.<init>(WebView.java:472)
	at android.webkit.WebView.<init>(WebView.java:459)
	... 69 more
```

--- a/build.gradle
+++ b/build.gradle
@@ -48,7 +48,7 @@ project.ext {
 
     // AndroidX
     annotationVersion = "1.1.0"
-    appcompatVersion = "1.1.0"
+    appcompatVersion = "1.2.0"
     mediaVersion = "1.1.0"
     preferenceVersion = "1.1.1"
     workManagerVersion = "2.3.4"

